From 542b0b0e2a4d15b2523efbdb8cc53ec3fc76570b Mon Sep 17 00:00:00 2001
From: Nicholas Hutchinson <nshutchinson@gmail.com>
Date: Sun, 2 Jun 2019 11:29:12 +0100
Subject: [PATCH] Fix build of leak_check target on MSVC

---
 absl/debugging/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/absl/debugging/CMakeLists.txt b/absl/debugging/CMakeLists.txt
index f66688b..4c1fc50 100644
--- a/absl/debugging/CMakeLists.txt
+++ b/absl/debugging/CMakeLists.txt
@@ -181,9 +181,9 @@ absl_cc_library(
   NAME
     leak_check
   HDRS
-    "$<$<NOT:$<CXX_COMPILER_ID:MSVC>>:leak_check.h>"
+    "leak_check.h"
   SRCS
-    "$<$<NOT:$<CXX_COMPILER_ID:MSVC>>:leak_check.cc>"
+    "leak_check.cc"
   COPTS
     ${ABSL_DEFAULT_COPTS}
   DEPS
-- 
2.21.0

